articles/provider-quantinuum.md

```diff
@@ -49,39 +47,58 @@ We recommend that users first validate their code using our API Validator. This
-
+Billing information: H1 Emulator usage is offered free-of-charge with a hardware subscription. For details, see Billing information for the [System Model H1](#system-model-h1).
 
-## System Model H1, Powered by Honeywell
+## System Model H1
 
-Quantinuum's quantum computer, the System Model H1, Powered by Honeywell, includes two hardware machine targets: H1-1 and H1-2.  Both machines have fundamentally the same design and both meet a nominal set of technical requirements. However, they may have system-to-system variability in exact performance and features, such as the maximum number of available qubits. Users are encouraged to test compatibility of their code by submitting jobs to the [API Validator](#api-validator) prior to submitting them to the target machines.  
+Quantinuum's quantum computer, the System Model H1, Powered by Honeywell, includes two hardware machine targets: H1-1 and H1-2.  Both machines have fundamentally the same design and both meet a nominal set of technical requirements. However, they may have system-to-system variability in exact performance and features, such as the maximum number of available qubits. Users are encouraged to test compatibility of their code by submitting jobs to the [Syntax Checker](#syntax-checker) prior to submitting them to the target machines.  
 
-Users may submit jobs to a specific machine (H1-1 or H1-2), or submit them to the *machine family*.  Submission to the machine family enables the submitted job to run on the first available, compatible machine. The only condition for compatibility is the number of qubits. If a user submits a job to a specific machine that is not available, the job will remain in that machine's queue until the machine becomes available.   
+Users may submit jobs to a specific machine (H1-1 or H1-2), or submit them to the *machine family*.  Submission to the machine family enables the submitted job to run on the first available, compatible machine. The only condition for compatibility is the number of qubits. If a user submits a job to a specific machine that is not available, the job will remain in that machine's queue until the machine becomes available.
 
 Both System Model H1 hardware H1-1 and H1-2 are continuously upgraded throughout their product lifecycle. Users are given access to the most up-to-date, advanced, and capable hardware available.
 
-
 - Job type: `Quantum Program`
 - Data Format: `honeywell.openqasm.v1`
 - Target ID:
-    - H1-1: `quantinuum.hqs-lt-s1` 
-    - H1-2: `quantinuum.hqs-lt-s2`
-    - H1 machine family: `quantinuum.hqs-lt`
+  - H1-1: `quantinuum.hqs-lt-s1` 
+  - H1-2: `quantinuum.hqs-lt-s2`
+  - H1 machine family: `quantinuum.hqs-lt`
 - Target Execution Profile: [Basic Measurement Feedback](xref:microsoft.quantum.target-profiles)
 
+Billing information:
```

Comment:
The billing information has been moved to the new [pricing page](https://docs.microsoft.com/en-us/azure/quantum/pricing?tabs=tabid-AQcredits%2Ctabid-AQcreditsQ%2Ctabid-payasgo&pivots=ide-computing#quantinuum), based on SEO and user needs. We try to keep providers' pages for technical details about the solvers.